Cargo.toml

```diff
@@ -20,7 +20,7 @@ exclude = ["/data"]
 rand = { version = "0.8", default-features = false }
 sha3 = { version = "0.10", default-features = false }
 kem = { version = "0.2", optional = true }
-zeroize = { version = "1.5.7", default-features = false, optional = true }
+zeroize = { version = "1.8", default-features = false, optional = true }
```

Comment:
Nitpick. But this upgrade still stays within the same semver compatible range (one dot whatever). Did you A) just want to pull in the latest `zeroize` because it's nice to have, or B) want to use features only available from `1.8` and up? If A, only the lockfile needs to be bumped (which you don't have one). Cargo will automatically pull in the latest `zeroize`, so anyone depending on this can easily get `1.8` without this change.

Reply:
Indeed. You mentioned the best practice. So I implemented it!

Reply:
Cool! Whether or not a library should commit their lockfile is a bit of a preference and can vary from project to project. I like to quite often commit them for reproducibility when issues are found.

My main point was that the dependency version spec should be somewhere along the lines of "lowest version that we compile and work with"